openstef/pipeline/optimize_hyperparameters.py

```diff
@@ -201,6 +201,11 @@ def optimize_hyperparameters_pipeline_core(
     report = objective.create_report(model=best_model)
 
     trials = objective.get_trial_track()
```

Comment:
It would be nice to add the trial fix in this function or class. Then we keep above code clean